uiplib/gui/mainGui.py

```diff
@@ -13,6 +13,10 @@
 from uiplib.settings import DEFAULT_FAVOURITE_PICS_FOLDER
 from uiplib.scrape import download
 from uiplib.uipImage import UipImage
+from uiplib.utils.utils import update_images
+from uiplib.utils.utils import update_pics_from_fav_pics
```

Comment:
because `update_pics_from_fav_pics` is not used in this module. remove this import